### ISSUE TITLE
Links on Mobile Screen Archetypes down

### ISSUE THREAD

**Issue by Chezlui** [codepath/android_guides#247]
cc: @nesquena 

In this page all the links related to screenshots are broken: http://guides.codepath.com/android/Mobile-Screen-Archetypes

I could edit this if help needed, but design is always highly personal xDD. Just let me know.

**Comment by cvoronin** [comment]
Those images (f. e. http://www.android-app-patterns.com/img/sets/settings/155_10_settings.png) are linked to www.android-app-patterns.com, it seems to be down now (ERR_NAME_RESOLUTION_FAILED for me)

**Comment by nesquena** [comment]
Yes this is linked to the larger issue of  http://www.android-app-patterns.com no longer resolving as of a few days ago. I am hoping it comes back up but if not, we'll have to replace all images. I will say they were getting outdated anyways. I would really appreciate any help replacing the screenshots with awesome material design examples of the archetypes

**Comment by Chezlui** [comment]
Google has data in Cache from Nov'16 so there may be a chance that comes alive in some moment. Is up to you @nesquena if you think it worth wait for it we can set a deadline. If not, I can do it easyly tomorrow.

**Comment by nesquena** [comment]
@Chezlui those examples are outdated now. They should all be replaced. Any help you can provide in doing that would be much appreciated!

**Comment by Chezlui** [comment]
@nesquena I will be delighted to do this next monday, I'm not sure if you closed it by mistake.

**Comment by nesquena** [comment]
@Chezlui Thanks! Yes I did close it by mistake.

**Comment by Chezlui** [comment]
Cool, I'll call you back when is done.

**Comment by Chezlui** [comment]
Ok, done.

It was more complicated than I thought. The references from the webpage https://pttrns.com/android-patterns is the most structured and fresher screenshots that I've found. They aren't brand new but on the other side I'm not sure if displaying too new designs is a good idea since they may not be too good pedagogically talking. 

Nonetheless I've added some references from http://androidniceties.tumblr.com/ that are "nice" indeed.
I'm sorry for not being able to have found a consistent source as the old android app patterns. In any case I hope it helps.

Kind regards
